skimage/morphology/binary.py

```diff
@@ -110,6 +140,15 @@ def binary_dilation(image, footprint=None, out=None):
     out : ndarray of bool, optional
         The array to store the result of the morphology. If None is
         passed, a new array will be allocated.
+    mode : str, optional
```

Comment:
This argument name caught me by surprise, since usually `mode` defines how boundary values are obtained: `{'reflect', 'constant', 'edge', 'symmetric', 'wrap'}`.

Reply:
It is exactly the same meaning, but the options are different. The ndimage function we call here only has the option of a constant value (you can either pad with 0s or with 1s). We’re using the strings ‘min’ and ‘max’ for those two options, they match the strings used in the gray-scale morphology. And we added the ‘ignore’ option (which we also added to the gray-scale version), which is ‘max’ for erosion and ‘min’ for dilation.

We cannot easily implement ‘reflect’ and the other more complex modes.  Though these are available in the gray-scale versions of the operators, which we could be using here instead. They all do the same thing for binary images anyway.